utils/create-component/createComponent.js

```diff
@@ -6,7 +6,6 @@ const path = require('path');
 const mkdirp = require('mkdirp');
 const inquirer = require('inquirer');
 const cmd = require('node-cmd');
-const colors = require('colors');
```

Comment:
This wasn't being used, so I removed it.

Reply:
@alanbsmith This is a special import. It extends the string prototype, but doesn't actually get called specifically. This allows us to do things like `'This is a warning'.yellow`, and it's used several times in this file. I'm pretty sure removing this will break that functionality.

Reply:
ohhh. Thanks for catching that! I'll put it back!